src/RolloutScorer/RolloutScorer/Utilities.cs

```diff
@@ -0,0 +1,119 @@
+using Microsoft.Azure.KeyVault.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+using Newtonsoft.Json;
+using Octokit;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Reflection;
+
+namespace RolloutScorer
+{
+    public class Utilities
+    {
+        public const string HotfixAzureDevOpsTag = "[HOTFIX]";
+        public const string RollbackAzureDevOpsTag = "[ROLLBACK]";
+        public const string IssueLabel = "Rollout Issue";
+        public const string HotfixLabel = "Rollout Hotfix";
+        public const string RollbackLabel = "Rollout Rollback";
+        public const string DowntimeLabel = "Rollout Downtime";
+
+        // The git file mode of a "file blob"; more info here: https://developer.github.com/v3/git/trees/#parameters
+        public const string GitFileMode = "100644";
+
+        public const string KeyVaultUri = "https://engkeyvault.vault.azure.net";
+        public const string GitHubPatSecretName = "BotAccount-dotnet-bot-repo-PAT";
+        public const string StorageAccountKeySecretName = "rolloutscorecards-storage-key";
+        public const string StorageAccountName = "rolloutscorecards";
+        public const string ScorecardsTableName = "scorecards";
+
+        public static bool IssueContainsRelevantLabels(Issue issue, string issueLabel, string repoLabel, ILogger log = null)
+        {
+            if (issue == null)
+            {
+                WriteWarning("A null issue was passed.", log);
+                return false;
+            }
+
+            return issue.Labels.Any(l => l.Name == issueLabel) && issue.Labels.Any(l => l.Name == repoLabel);
+        }
+
+        /// <summary>
+        /// Parse and return config
```

Comment:
This comment doesn't add much. :-)  It's almost a word for word restatement of the method name and return type.

Reply:
Removed